tokio/src/runtime/handle.rs

```diff
@@ -16,7 +16,11 @@ use std::{error, fmt};
 #[derive(Debug, Clone)]
 pub struct Handle {
     pub(super) spawner: Spawner,
+}
 
+/// All internal handles that are *not* the scheduler's spawner.
+#[derive(Debug)]
+pub(crate) struct HandleInner {
```

Comment:
nit, take it or leave it: as an alternative to `HandleInner`, what if we called this `Resources` or something, since it's mostly resource drivers? i guess the blocking spawner isn't a resource driver...

Reply:
It isn't really resources, it is `Handle` - task scheduler handle. "Inner" isn't great, but it is an established pattern. I rather stick w/ convention unless there is a reason not to.